atd-vze/src/queries/gqlAbstract.js

```diff
@@ -222,7 +234,7 @@ gqlAbstractTableAggregateName (
       output.push(`where: {${where.join(", ")}}`);
     }
 
-    if (this.config["order_by"] !== null) {
+    if (this.config["order_by"]) {
```

Comment:
probably not necessary but you can do the double-bang trick here if you want to be confident that you'll have a truthy value.